fibonacci/csharp/code.cs

```diff
@@ -0,0 +1,18 @@
+﻿static int Fibonacci(int n) {
+    if (n == 0) {
+        return 0;
+    }
+
+    if (n == 1) {
```

Comment:
Maybe combine the if statement into one?
```cs
if (n <= 1)
    return n;
```

Reply:
Good point, done